test/node/test_front.rb

```diff
@@ -59,4 +59,35 @@ def test_renders_public_pages
       assert(json['all'].find { |r| r['host'] == 'localhost' })
     end
   end
+
+  def test_different_logos
+    FakeNode.new(log: $log).run(['--ignore-score-weakness']) do |port|
+      {
+        '0' => 'https://zold.io/logo-red.png',
+        '4' => 'https://zold.io/logo-orange.png',
+        '16' => 'https://zold.io/logo-green.png'
+      }.each do |num, path|
+        score = Zold::Score.new(
+          Time.now, 'localhost', 999,
+          'NOPREFIX@ffffffffffffffff',
+          strength: 1
+        )
+
```

Comment:
@oltdaniel get rid of empty lines inside method bodies, see: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
Will be done tomorrow after 4PM German time, when school is finished.